External/FEXCore/Source/Interface/Core/Dispatcher/Dispatcher.cpp

```diff
@@ -655,17 +656,22 @@ void Dispatcher::RemoveCodeBuffer(uint8_t* start_to_remove) {
   }
 }
 
-bool Dispatcher::IsAddressInJITCode(uint64_t Address, bool IncludeDispatcher) const {
+bool Dispatcher::IsAddressInJITCode(uint64_t Address, bool IncludeDispatcher, bool IncludeCompileService) const {
   for (auto [start, end] : CodeBuffers) {
     if (Address >= start && Address < end) {
       return true;
     }
   }
 
+  bool Res = false;
```

Comment:
How about this to clarify the return value strategy?
```
  if (IncludeDispatcher && IsAddressInDispatcher(Address)) {
      return true;
  }
  
  if (IncludeCompileService && ThreadState->CompileService &&
        ThreadState->CompileService->ISAddressInJITCode(Address)) {
     return true;
  }
  
  return false;
```

Reply:
Changed to this example.